WordPress/WordPressTest/SiteCreation/SiteSegmentsCellTests.swift

```diff
@@ -45,7 +45,18 @@ final class SiteSegmentsCellTests: XCTestCase {
     }
 
     func testCellTitleIsTheCorrectFont() {
-        XCTAssertEqual(cell?.title.font, WPStyleGuide.fontForTextStyle(.body, fontWeight: .semibold))
+        // SiteSegmentsCell's UILabel has `title.adjustsFontForContentSizeCategory = true`.
+        //  We cannot relaly assert on the Label's font, since it's bond to be changed.
+        //
+        //  XCTAssertEqual(cell?.title.font, WPStyleGuide.fontForTextStyle(.body, fontWeight: .semibold))
+
+        let expectedFont = WPStyleGuide.fontForTextStyle(.body, fontWeight: .semibold)
```

Comment:
Does it make more sense to be able to pass in the trait collection to `WPStyleGuide.fontForTextStyle` for testing purposes?

Would end up with something like this:
```swift
WPStyleGuide.fontForTextStyle(.body, fontWeight: .semibold, compatibleWith: cell?.title.traitCollection)
```

Reply:
@bjtitus Actually, now that you mention it, we might be able to use [this API](https://developer.apple.com/documentation/uikit/uitraitcollection/3238082-performascurrent) to prettify things up a tiny bit.

That being said, personally speaking, IMO, I would remove the test entirely. Unit Testing that a font is set by our UITableViewCell method overrides isn't really valuable (again, IMO!).

Thank you sir!!